packages/core-flows/src/definition/cart/steps/create-order-from-cart.ts

```diff
@@ -0,0 +1,57 @@
+import { CartWorkflowDTO } from "@medusajs/types"
+import { OrderStatus } from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+import { createOrdersWorkflow } from "../../../order/workflows/create-orders"
+
+interface StepInput {
+  cart: CartWorkflowDTO
+}
+
+export const createOrderFromCartWorkflowId = "create-order-from-cart"
+export const createOrderFromCartWorkflow = createStep(
```

Comment:
question: Should this be suffixed with `Step` instead of workflow?

If this is intended to be a workflow, then we would use `createWorkflow` and `runAsStep` in cart-completion

Reply:
good catch, will move it around.